source/frontend/src/__tests__/GoogleDocsContext.test.js

```diff
@@ -0,0 +1,11 @@
+const useGoogleDocs = require("../GoogleDocsContext");
+const GoogleDocsContext = require("../GoogleDocsContext")

```

Comment:
Just note that this depends on the directory structure.

Otherwise looks good.